core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/operators/ArithmeticUnitTest.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ArithmeticUnitTest {
+
+    @Test
+    void whenUseTheOperatorArithmeticAdditionToConcatenate() {
+
```

Comment:
We don´t need this extra blank space. The only one needed is between sections (given, when, then).